protocols/interfaces.go

```diff
@@ -44,6 +45,22 @@ func NewWithdrawAllTransaction(channelId types.Destination, signedState state.Si
 	return WithdrawAllTransaction{SignedState: signedState, ChainTransaction: ChainTransactionBase{channelId: channelId}}
 }
 
+type ChallengeTransaction struct {
+	ChainTransaction
+	Candidate     state.SignedState
+	Proof         []state.SignedState
+	ChallengerSig crypto.Signature
+}
+
+func NewChallengeTransaction(
+	channelId types.Destination,
+	candidate state.SignedState,
+	proof []state.SignedState,
+	challengerSig crypto.Signature,
+) ChallengeTransaction {
+	return ChallengeTransaction{ChainTransaction: ChainTransactionBase{channelId: channelId}, Candidate: candidate, Proof: proof, ChallengerSig: challengerSig}
```

Comment:
This is extremely picky (and a preference) but would you consider breaking this up over multiple lines? At least for me it makes it a lot easier to parse.

Reply:
[5d563b5](https://github.com/statechannels/go-nitro/pull/1619/commits/5d563b5dd2cc34ac57fbe234262046abdc3b31d4)